style.css

```diff
@@ -86,3 +92,179 @@ h1 span {
 .Menu {
   padding-right: 30px;
 }
+
+h2 {
+  margin-bottom: 20px;
+  width: 327px;
+  height: 97px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.Works {
+  margin-top: 62px;
+  width: 375px;
+  display: grid;
+  row-gap: 24px;
+  grid-template-columns: auto;
+  grid-template-rows: auto auto auto auto auto auto;
+  justify-content: center;
+  align-items: center;
+}
+
+.project {
+  width: 327px;
+  height: 474px;
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.project ul {
+  width: 305px;
+  height: 32px;
+  display: flex;
+  padding-inline-start: 0;
+  justify-content: space-between;
+  align-items: center;
+}
+
+h3 {
+  width: 295px;
+  height: 57px;
+  font-size: 24px;
+  line-height: 32px;
+  text-align: center;
+  vertical-align: center;
+  color: #3a4a42;
+}
+
+.tag {
+  padding: 8px 12px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+  list-style: none;
+}
+
+.aboutMe {
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.aboutMe h2 {
+  width: 375px;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.GetMyResume {
+  padding-top: 24px;
+  margin-bottom: 59px;
+  width: 375px;
+  display: flex;
+}
+
+.ilu1 {
+  padding-left: 24px;
+}
+
+.ilu2 {
+  padding-right: 40px;
+}
+
+.LFS {
+  width: 375px;
+  display: grid;
+  row-gap: 24px;
+  grid-template-columns: auto;
+  grid-template-rows: auto auto auto;
+  justify-content: center;
+  align-items: center;
+}
+
+.AboutMe2 {
+  width: 327px;
+  height: 352px;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.AboutMe2 img {
+  border: solid 16px;
+  border-radius: 50%;
+  border-color: #fff;
+  margin: 20px;
+  margin: auto;
+}
+
+.TagBox ul {
+  width: 320px;
+  height: 136px;
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  padding-inline-start: 0;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.AboutMe2 h3 {
+  margin: 0;
+  width: 295px;
+  height: 32px;
+  font-size: 24px;
+  line-height: 32px;
+  text-align: center;
+  vertical-align: center;
+  color: #344563;
+}
+
+.tag2 {
+  padding: 8px;
+  background: #fff;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #36b37e;
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible as per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) ,

  - Your design:
  
![yours](https://user-images.githubusercontent.com/97834160/205418798-35305fdb-63fc-4774-8ca2-7361992c2919.png)

  - Figma design:
![skillsF](https://user-images.githubusercontent.com/97834160/205418804-59fa6b4c-7413-479b-97ec-3cac59d3a1ae.png)

Reply:
@GedeonTS I will check this, about the OPTIONAL part. I have the background on my local repository. But for some reason when I git push it doesn't go to the Repository on GitHub. Do you why does it happen?

![background](https://user-images.githubusercontent.com/26579035/205424749-d21ffa08-6232-4a03-8aca-1757abb04df8.JPG)
![background_repo](https://user-images.githubusercontent.com/26579035/205424756-5791d1a9-37ae-4aae-b30d-97d070e3c313.JPG)